frontend/client/styles/components/navbar.scss

```diff
@@ -1,23 +1,39 @@
 @import '../color.scss';
+@import '../include-media';
 
 .navbarContainer {
   display: flex;
   flex-direction: row;
   justify-content: space-between;
   align-items: center;
-  padding: 10px 98px 10px 101px;
+  @include media('>phone') {
+    padding: 10px 98px;
+  }
+  @include media('<=phone') {
+    padding: 2px 10px;
+  }
   border-bottom: 1px solid $lightGray;
 
   .logo-link {
     display: flex;
   }
 
   #orgLogo {
-    height: 40px;
+    @include media('>phone') {
+      height: 40px;
+    }
+    @include media('<=phone') {
+      height: 30px;
+    }
   }
 
   .menu-btn {
-    margin-right: 88px;
+    @include media('>phone') {
+      margin-right: 88px;
```

Comment:
Do we need the navbar to be pushed over for any particular reason? It kinda looks wonky to me like that, but if it's a design choice then nevermind.

Reply:
IIRC, we added that padding to try and set a maximum width for the app. According to git, you were the one who added that line ;-)

What aspect of it looks "wonky"? Can you post a screen shot? It could have something to do with different screen widths.

Reply:
This is a valid concern but not relevant to these changes, so let's push it off to another issue/PR if there's a better way to handle this